### PR TITLE
Release v4.2.1

### PR DESCRIPTION
**Pull request by dvaldivia**
Signed-off-by: Daniel Valdivia <18384552+dvaldivia@users.noreply.github.com>

### CODE REVIEW

**Review by harshavardhana — APPROVED** [review]

**Review by harshavardhana — CHANGES_REQUESTED** [review]
please fix the conflicts @dvaldivia

**Comment by dvaldivia** [comment]
done @harshavardhana

File: index.yaml [hunk 1]
```diff
@@ -2,7 +2,7 @@ apiVersion: v1
 entries:
   minio-operator:
   - apiVersion: v2
-    appVersion: v4.2.0
+    appVersion: v4.2.1
```
Comment (harshavardhana) [inline index.yaml:5]: Don't edit this yourself @dvaldivia this is generated automatically by `helm-reindex.sh`

**Review by harshavardhana — CHANGES_REQUESTED** [review]

File: index.yaml (hunk 1 above)
Comment (dvaldivia) [reply index.yaml:5]: was not aware, just ran it, it updated the digest and updated a tar, is it supposed to do that?

Comment (harshavardhana) [reply index.yaml:5]: Yes

File: README.md [hunk 2]
```diff
@@ -145,7 +145,7 @@ version](https://github.com/minio/operator/releases/) of the MinIO
 Kubernetes Plugin:
 
 ```sh
-wget https://github.com/minio/operator/releases/download/v4.2.0/kubectl-minio_4.2.0_linux_amd64 -O kubectl-minio
+wget https://github.com/minio/operator/releases/download/v4.2.1/kubectl-minio_4.2.0_linux_amd64 -O kubectl-minio
```
Comment (harshavardhana) [inline README.md:148]:
```suggestion
wget https://github.com/minio/operator/releases/download/v4.2.1/kubectl-minio_4.2.1_linux_amd64 -O kubectl-minio
```

**Review by harshavardhana — CHANGES_REQUESTED** [review]

**Review by harshavardhana — APPROVED** [review]

**Pull request merged**